opensearch/src/test/java/org/opensearch/sql/opensearch/data/type/OpenSearchDataTypeTest.java

```diff
@@ -104,8 +104,8 @@ private static Stream<Arguments> getTestDataWithType() {
         Arguments.of(MappingType.ScaledFloat, "scaled_float", DOUBLE),
         Arguments.of(MappingType.Double, "double", DOUBLE),
         Arguments.of(MappingType.Boolean, "boolean", BOOLEAN),
-        Arguments.of(MappingType.Date, "date", TIMESTAMP),
-        Arguments.of(MappingType.DateNanos, "date", TIMESTAMP),
+        Arguments.of(MappingType.Date, "timestamp", TIMESTAMP),
```

Comment:
Please double confirm on UT chagne. does all these date -> timestamp change necessary?

Reply:
All these assertions directly comparing with `legacyTypeName` and hence need this update. I think these changes are necessary and only in UT to avoid any breaking changes with type change from `ExprCoreType` to `OpenSearchDateType` inside `fieldMappings` map.

Reply:
I only find the legacy name change for `Date` in production code, why the `DateNanos` changed in test either?

Reply:
I think this is because when we create `OpenSearchDataType` instance for every field mapping type, we are creating `OpenSearchDateType` instance for both `Mappingtype.Date` and `MappingType.DateNanos`- https://github.com/opensearch-project/sql/blob/main/opensearch/src/main/java/org/opensearch/sql/opensearch/data/type/OpenSearchDataType.java#L165 
and set the mapping type to Date even for DateNanos (not sure why is it this way) - https://github.com/opensearch-project/sql/blob/main/opensearch/src/main/java/org/opensearch/sql/opensearch/data/type/OpenSearchDateType.java#L143
So I think here UT assertion still looking for Mapping type date even for date nanos I believe.